.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig : http://EditorConfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
I wish editorconfig could point to the copy in mofo-style to avoid copy pasta, but I think the only way to do that is a symlink, which isn't kosher for Windows.